.vscode/settings.json

```diff
@@ -18,4 +18,9 @@
     "workbench.editorAssociations": {
         "*.ipynb": "jupyter-notebook"
     },
+    "cSpell.words": [
+        "allometric",
+        "coef",
+        "Testasaurus"
+    ],
```

Comment:
Was this intentionally committed?

Reply:
Haha no it was not, that's pretty funny though. I was tired of seeing them in my local space, clicked for what I thought was visual studio to learn the words and stop highlighting them as spelling errors. Best way to undo that?

Reply:
That's what it has done, but it stores the words in the same file we use to coordinate settings. Maybe there is an alternative location for spelling?

Reply:
Fixed!

Reply:
Sorry I'm so late to the party, but, yes, @davidorme there is an alternative location for saving spellings. You just have to choose "add to user settings" rather than "add to workspace settings", which makes it global.